src/containers/home/index.js

```diff
@@ -32,17 +29,15 @@ class Home extends Component {
     decrementAsync: PropTypes.func.isRequired,
     changePage: PropTypes.func.isRequired,
   };
-
+  static defaultProps = {
+    dataCar: null,
```

Comment:
If you define an inital state in redux you should make this an required prop and not define default prop.